### DISCUSSION TITLE
Option for missions to hide on completing

### DISCUSSION THREAD

**Discussion by Davisiiiik** [BG-Software-LLC/SuperiorSkyblock2#962] [Pending Feature Requests] [answered]
Hello, I've got around 200 missions (in 16 categories) and a lot of them are done like two different missions. First one is non-repeatable and the reward is big, second one is repeatable with lower reward. They are visually named and described the same. I show to player repeatable mission only after he completes the first non-repeatable, so I have a lot of pairs of the same mission.

So my feature request is: Can you add option for every mission type (ie: boolean `hide-on-completition` with default value `false`), that would hide the mission from mission category on completing when `hide-on-completition` option is in `true` state?

**Comment by Lemvs** [comment, marked as answer]
remove-completed: true in menus/missions-category.yml

@OmerBenGera you can mark this answered and close it,
